app/controllers/api/v1/planes_reservations_controller.rb

```diff
@@ -1,5 +1,12 @@
 class Api::V1::PlanesReservationsController < ApplicationController
-  def index; end
+  def index
+    reservations = PlaneReservation.where(user_id: @current_user.id).includes(:plane)
+    reservations = reservations.map do |reservation|
+      { reservation:, image_url: reservation.plane.image }
+    end
+
+    render json: reservations
+  end
```

Comment:
OPtional: I think this is the @ginabeki  part, I think it will be better to calculate duration when we give it back instead of saving it in the database. It is consuming the database unnecessarily.